src/TableTransforms.jl

```diff
@@ -12,6 +12,8 @@ using LinearAlgebra
 using Statistics
 using PrettyTables
 using CategoricalArrays
+using Random
+using StatsBase
```

Comment:
```suggestion
using StatsBase: AbstractWeights
```
Also, please move this line to before LinearAlgebra given that it is not a stdlib.